eng/update-testhost-gcserver.fsx

```diff
@@ -0,0 +1,124 @@
+// Windows-only utility for F# build: ensures <gcServer enabled="true"/> in .NET Framework testhost configs.
+// - Prints local .dotnet SDK version
+// - Updates BOTH AnyCPU and x86 testhost.net472*.exe.config files under the local .dotnet SDK
+// - Intended to run ONCE before desktop/VS tests (not per test)
+// Usage (from repo root): dotnet fsi eng/update-testhost-gcserver.fsx
+
+open System
```

Comment:
Is the reason we need this the fact that for .netframework `SET DOTNET_gcServer=1` does do anything, and `.config` files are really the only option there is? :((

Modifying the fresh SDK installation is not something I would like to maintain (at least luckily, this is only for an isolated test project run)

Reply:
From what I tried there is no other working way to set server gc for desktop. This speeds up the testDesktop run considerably locally but unfortunately doesn't do much for CI speeds that regressed again :(

I'll revert it.